deploy/deploy-operator-yaml/Pulumi.yaml

```
@@ -0,0 +1,13 @@
+name: pulumi-operator-yaml
+runtime: yaml
+description: |
+  Deploys the Pulumi Kubernetes Operator.
+config:
+  version: # The version to install of the Pulumi Kubernetes Operator.
+    type: string
+    default: v2.0-devel
+resources:
+  pko:
+    type: kubernetes:kustomize/v2:Directory
+    properties:
+      directory: https://github.com/pulumi/pulumi-kubernetes-operator//operator/config/default/?ref=${version}
```

nit: remove extra `/`
```suggestion
      directory: https://github.com/pulumi/pulumi-kubernetes-operator/operator/config/default/?ref=${version}
```

---

The double-slash is by design: https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md#remote-directories

>  The directory is specified by appending a // after the repo URL.